CHANGELOG.rst

```diff
@@ -29,10 +29,13 @@ Changed
 '''''''
 - simplify ``op`` decorator
 - simplify default implementation of ``Op.__repr__``
+- complete rewrite of the readme
```

Comment:
I do not think that this needs to be in the release notes.

Reply:
Why not? There's no reason to limit the changelog to pure code.

Reply:
That's true. But does it add important information for the user? I see two types of users:
a) A new user will have a look at the README, install the package and will read the documentation in order to learn usage of the framework
b) An existing user has already installed and used the package, he will look at the CHANGELOG to see which changes will or might potentially affect his usage of the framework. A change in README is not important for him, unless there are important changes mentioned in the README and not in the CHANGELOG - which would mean that the CHANGELOG should be  'changed'.

If you want to mention it just add "Extended documentation with examples highlighting usage of the new API, cf. README"